applications/CableNetApplication/python_scripts/apply_weak_sliding_process.py

```diff
@@ -0,0 +1,41 @@
+import KratosMultiphysics as KratosMultiphysics
+import KratosMultiphysics.CableNetApplication as CableNetApplication
+
+def Factory(settings, Model):
```

Comment:
I think this can be simplified a lot, take a look at what I did [here](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/StructuralMechanicsApplication/python_scripts/postprocess_eigenvalues_process.py)

Reply:
will you change this?